presto-main/src/main/java/com/facebook/presto/execution/scheduler/SqlQueryScheduler.java

```diff
@@ -130,7 +148,11 @@ public static SqlQueryScheduler createSqlQueryScheduler(
             Session session,
             QueryStateMachine queryStateMachine,
             SubPlan plan,
-            boolean summarizeTaskInfo)
+            boolean summarizeTaskInfo,
+            List<PlanOptimizer> runtimePlanOptimizers,
+            PlanNodeIdAllocator idAllocator,
```

Comment:
nit: let's group both, the `PlanNodeIdAllocator ` and the `PlanVariableAllocator` together